.github/workflows/pr-build-live-branch.yml

```diff
@@ -0,0 +1,123 @@
+name: Build live branch for PR
+
+on:
+  pull_request:
```

Comment:
May change this (equal to manual triggers) and we will slowly roll out this workflow to all PRs later? 

```suggestion
  workflow_dispatch:
```

Reply:
I agree, manual should be fine to begin with and later we can discuss about creating it for every PR

Reply:
Yeah, it turns out that I still need the PR number to inform Jetpack Beta Builder so I think it would be better to use `pull_request` context. 

But more important as discussed in https://github.com/Automattic/woocommerce-payments/pull/5768#discussion_r1138451184, we can bring the benefit right away with the new comment including the link to spin up a new JN site right away.